firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -34,14 +34,16 @@ public expect val Firebase.firestore: FirebaseFirestore
 /** Returns the [FirebaseFirestore] instance of a given [FirebaseApp]. */
 public expect fun Firebase.firestore(app: FirebaseApp): FirebaseFirestore
 
-public expect class NativeFirebaseFirestore
+internal expect class NativeFirebaseFirestore
 
 public class FirebaseFirestore internal constructor(private val wrapper: NativeFirebaseFirestoreWrapper) {
 
-    public constructor(native: NativeFirebaseFirestore) : this(NativeFirebaseFirestoreWrapper(native))
+    public companion object {}
```

Comment:
To circumvent https://github.com/GitLiveApp/firebase-kotlin-sdk/pull/555/files#r1655371860 we introduce a companion object